src/cpp/src/visual_language/vision_encoder.cpp

```diff
@@ -363,28 +363,21 @@ ProcessorConfig from_any_map(
     return extracted_config;
 }
 
-
-ov::Tensor preprocess_image_llava(const ov::Tensor& image, const ProcessorConfig& config) {
+// TODO Consider moving to clip.cpp or separate dedicated file
```

Comment:
That isn't something that is hard to do. Either do it now or remove the comment. Same below

Reply:
As we were planning to refactor, removing for now